homer0/projext-plugin-webpack
# webpack 4

### What does this PR do?

Yes, finally!! This PR updates all the dependencies for webpack 4.

This is breaking because most dependencies were updated to their latest version.

### How should it be tested manually?

This _"shouldn't be breaking"_, so just install it on a project you were using the plugin with and it should build and run as usual.

And of course...

```bash
npm test
# or
yarn test
```
